internal/app/mmlogic/mmlogic_service.go

```diff
@@ -96,12 +96,12 @@ func getPageSize(cfg config.View) int {
 
 	pSize := cfg.GetInt("storage.page.size")
 	if pSize < minPageSize {
-		logger.Warningf("page size %v is lower than the minimum limit of %v", pSize, maxPageSize)
+		mmlogicServiceLogger.Infof("page size %v is lower than the minimum limit of %v", pSize, maxPageSize)
```

Comment:
Is there any way to avoid this message? This message will be logged every time when queryticket is called with invalid page size regardless of doing it with warning or info level.

Reply:
Rejected out of bounds requests is a possibility here. Not sure why this is coercing the values.